src/main/common/router/routablePath.ts

```diff
@@ -30,4 +32,21 @@ export class RoutablePath {
     const viewPath: string = split.slice(isCaseUri ? 2 : 1).join('/')
     return `${featureName}/views/${viewPath}`
   }
+
+  evaluateUri (substitutions: {[key: string]: string}): string {
+    if (substitutions === undefined || Object.keys(substitutions).length === 0) {
+      throw new Error('Path parameter substitutions are required')
+    }
+
+    const path = Object.entries(substitutions).reduce((uri: string, substitution: [string, string]) => {
+      const [parameterName, parameterValue] = substitution
+      return uri.replace(`:${parameterName}`, parameterValue)
```

Comment:
Would this work?
```
const replacedValue = uri.replace(`:${parameterName}`, parameterValue)
return replacedValue !== uri ? replacedValue : throw new Error('Didn’t find match')
```

Reply:
Some other options on here:
https://stackoverflow.com/questions/5257000/how-to-know-if-javascript-string-replace-did-anything

Reply:
Logically should work.

Reply:
Done it this way. Not exactly the syntax but logic the same